<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepositMoney</class>
 <widget class="QDialog" name="DepositMoney">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #4a4a4a;
}

QStackedWidget {
    background-color: #4a4a4a;
    color: #ffffff;
}

QPushButton {
background-color: gray;
border-radius: 10px;
color:lightblue;
}

QLabel#label_main_menu,
QLabel#label_transfers,
QLabel#label_withdrawal_deposit {
    font: 20pt &quot;Arial&quot;;
    color: lightyellow;
}

QLabel {
color:lightblue;
font-family: Arial;
font-size: 12pt;
}

QLineEdit {
font-family: Arial;
font-size: 10pt;
background-color: lightgray;	
border: 1px solid white;
border-radius: 10px;
}

QTableView {
    background-color: #2e2e2e;
    color:lightblue;
border-radius: 10px;
}

QHeaderView::section {
    background-color: #1f1f1f;
	color:lightblue;
}

QTableView::item {
    border: 1px solid #3a3a3a;
}

QTableView::item:selected {
    background-color: #4a4a4a;
	color:lightblue;
 	gridline-color: gray
}
QHeaderView:vertical {
    background-color: #1f1f1f;
	color:lightblue;
    border: 1px solid #3a3a3a;
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>300</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_choosing_amount">
    <widget class="QPushButton" name="btn_deposit">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>150</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Deposit</string>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>461</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Amount:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="txt_amount"/>
      </item>
     </layout>
     <zorder>txt_amount</zorder>
     <zorder>label</zorder>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>471</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">    font: 15pt &quot;Arial&quot;;
    color: lightyellow;</string>
     </property>
     <property name="text">
      <string>Deposit an amount between 0 and 5000:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../BankingAppImages/Logo_Def.png</pixmap>
     </property>
    </widget>
    <zorder>formLayoutWidget</zorder>
    <zorder>label_6</zorder>
    <zorder>label_2</zorder>
    <zorder>btn_deposit</zorder>
   </widget>
   <widget class="QWidget" name="page_password">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>349</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">    font: 15pt &quot;Arial&quot;;
    color: lightyellow;</string>
     </property>
     <property name="text">
      <string>Enter password:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txt_password">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>349</width>
       <height>26</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_deposit_money">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>130</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Deposit Money</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../BankingAppImages/Logo_Def.png</pixmap>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
